tasks/release.rb

```diff
@@ -44,6 +44,13 @@
       raise "Could not insert PRE in #{file}" unless $1
 
       File.open(file, 'w') { |f| f.write ruby }
+
+      if File.exist?("#{framework}/package.json")
+        Dir.chdir("#{framework}") do
+          version = version.gsub(/\./).with_index { |s, i| i >= 2 ? '-' : s }
```

Comment:
@rafaelfranca see https://github.com/rails/rails/pull/24903#discussion_r62425201

@maclover7 Again, a helper method would eliminate this question ^ for anyone reading the code.